templates/web/fixmystreet.com/report/_cobrand_ref_oxfordshire.html

```diff
@@ -1,18 +1,16 @@
 [% reference = problem.get_extra_metadata('customer_reference') ~%]
-[% feature_id = problem.get_extra_field_value('feature_id') ~%]
-[% unit_number = problem.get_extra_field_value('unit_number') ~%]
 [% unit_type = problem.get_extra_field_value('unit_type') ~%]
 [% column_no = problem.get_extra_field_value('column_no') ~%]
 
 <div class="council_info_box">
     <p><strong>Council ref:</strong> [% reference OR problem.id %]</p>
-  [% IF feature_id OR unit_number %]
-    <p><strong>Asset ID:</strong> [% feature_id OR unit_number %]</p>
-  [% END %]
-  [% IF column_no %]
+    [% asset_id = problem.public_asset_id; IF asset_id %]
+        <p><strong>Asset ID:</strong> [% asset_id %]</p>
+    [% END %]
+    [% IF column_no %]
```

Comment:
No need to change it back, but note this was deliberate indentation on our part, not a mistake; it can be quite nice in templates to have HTML indented in 4, and the logic to also be indented in 4 but offset by 2, which makes it easy to follow when as here there are conditional and non-conditional parts of the HTML output.